### PR TITLE
perl6: Add more file name extensions

### PR DESCRIPTION
**Pull request by techee**
See https://en.wikipedia.org/wiki/Raku_(programming_language)

### CODE REVIEW

**Comment by masatake** [comment]
Private NOTE: https://docs.raku.org/language/pod

File: parsers/perl6.c [hunk 1]
```diff
@@ -325,7 +325,8 @@ findPerl6Tags (void)
 parserDefinition *
 Perl6Parser (void)
 {
-    static const char *const extensions[] = { "p6", "pm6", "pm", "pl6", NULL };
+    static const char *const extensions[] = { "p6", "pm6", "pm", "pl6",
+		"pod6", "t6", "raku", "rakumod", "rakudoc", "rakutest", NULL };
```
Comment (masatake) [inline parsers/perl6.c:329]: In  my understanding pod6 and rakumod are for extensions for a documentation language, not for Perl6.
I guess our perl6 parser doesn't handle input fils for the documentation language.

**Comment by techee** [comment]
Yes, you are right, I just removed it (together with `rakudoc` which is the same).

**Comment by masatake** [comment]
Thank you.

**Pull request merged**